docs/gettingStarted/setupAlchemyDevMode.md

```diff
@@ -1 +1,168 @@
-Coming Soon
+## Prerequisites
+
+  - docker >= 18.06.1-c
+  - docker-compose >= 1.22.0
+  - node >= 10.16.0
+  - npm >= 6.9.0
+  
+## Overview
+  Alchemy uses [Client.js](../../stack/client/clientIntro) for 
+  
+  - reading/inferencing blockchain data via [DAOstack Subgraph](../../stack/subgraph/subgraphIntro)
+  - writing/modifying state of [Arc contracts](../../stack/arcIntro/)
+
+  Interaction of Alchemy with rest of the stack
+  ![Interaction](../images/daostack-interaction.png)
+
+  Following is the guide to start developing with alchemy if you are using already supported schemes by `client.js` and `subgraph`. If you have created your own `scheme contracts` for your DAO, please refer to [Add Custom Scheme support to Alchemy](../customSchemeSupportAlchemy)
+
+## Boilerplate
+
+    git clone https://github.com/daostack/alchemy.git
+    cd alchemy
+    npm ci
+
+## Setup Alchemy with Ganache (mode: development)
+
+    docker-compose build
+    docker-compose up -d
+
+  The above commands will build docker images and start the following services locally:
+
+  - **alchemy** => react frontend with webpack-dev-server
+  - **alchemy-server** => for storing proposal information for quick access
+  - **graph-node** => for handling events from blockchain as described in subgraph
+  - **ganache** => dev blockchain with some test DAOs deployed and loaded with GEN and Eth
+  - **subgraph-ipfs** => subgraph mappings on ipfs node
+  - **subgraph-postgres** => db for caching events based on subgraph and later fetched via GraphQL
+  - **redis**
+  - **alchemy-postgres**
+
+  Import test accounts that are setup with GEN and ETH to your metamask. You can get the account details by:
+
+    docker logs alchemy_ganache_1 | head -35
+
+
+  Now your playground is ready for developing.
```

Comment:
This will work, but I prefer to run the alchemy server on the host machine (instead of in the docker container), as described here: https://github.com/daostack/alchemy/blob/dev/docs/development.md (and that will also avoid the TODO problem)

Reply:
I always run the server outside of the docker container.  This is particularly handy when flipping between ganache, staging and main networks.

Reply:
(do we document how to run against staging and main?)

Reply:
hah, see below

Reply:
its a bit confusing to use the word "server" like this but not refer to what we call "alchemy-server". how can we differentiate? Ive been calling them alchemy client and alchemy server.

Reply:
but yes I agree I always run alchemy client outside of docker, because you can use start, start-staging or start-prod to run against different networks

Reply:
Oh... I'm confused are you talking about `alchemy` or `alchemy-server` outside docker?

I saw start, start-staging and start-prod for alchemy